### PR TITLE
Pbm 435 doc pbm status

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

File: doc/source/running.rst [hunk 1]
```diff
@@ -66,8 +66,12 @@ The following diagram illustrates the backup flow.
 Checking an in-progress backup
 --------------------------------------------------------------------------------
 
+.. important::
+
+   As of version 1.4.0 the information about running backups is not available in the :command:`pbm list` output. Use the :command:`pbm status` command instead to check for running backups. See :ref:`pbm-status` for more information.
+
 Run the |pbm-list| command and you will see the running backup listed with a
-'In progress' label. When that is absent the backup is complete.
+'In progress' label. When that is absent, the backup is complete.
```
Comment (akira-kurogane) [inline doc/source/running.rst:74]: This is no longer the case I thought.

File: doc/source/status.rst [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+To check |PBM| status, run the following command:
+
+.. code-block:: bash
+
+   $ pbm status
+
+The output of the :command:`pbm status` command provides the information about:
+
+- Your MongoDB deployment and ``pbm-agents`` running in it: to what ``mongod`` node each agent is connected, the |pbm| version it runs and the agent's state
+- Running backups / restores, if any
```
Comment (akira-kurogane) [inline doc/source/status.rst:18]:
```suggestion
- The currently running backups / restores, if any
```

**Review by akira-kurogane — CHANGES_REQUESTED** [review]
Hi Anastasia.

"pbm list" will from 1.4 only list completed backups, so the "Checking an in-progress backup" section of running.rst should no longer advise that you might see an "In progress" label.

**Review by akira-kurogane — APPROVED** [review]
Hi @nastena1606. Thanks for change.

**Review by sandraromanchenko — APPROVED** [review]

**Review by dAdAbird — APPROVED** [review]

**Pull request merged**